docs/manage/enabling_surveys_using_config_files.md

```diff
@@ -0,0 +1,537 @@
+# Enabling Surveys using Config Files
+---
+
+The config files found in e-mission/nrel-openpath-deploy-configs are the basis for adding survey support for buttons
+
+## Before we begin:
+
+If you are going to create/test your own version of the config files, you must make sure that your config file will be read in by e-mission-phone by doing the following:
+
+_From the `e-mission-phone` root directory, navigate to `www/js/config/dynamic_config.js`. From here in the `DynamicConfig` Factory, there is a function named readConfigFromServer which contains the declaration for variable `const downloadURL = "..."`_ **Change the value of `downloadURL` to be the filepath for your config directory**
+```javascript
+// The URL prefix from which config files will be downloaded and read.
+// Change this if you supply your own config files.
+const downloadURL = "https://raw.githubusercontent.com/sebastianbarry/nrel-openpath-deploy-configs/surveys-info-and-surveys-data/configs/"+label+".nrel-op.json"
+```
+
+The `label` variable used by `downloadURL` is defined by the `label=` field in the "config login link" you use during the config login screen (Screenshot here). **Create a "config login link" using this format (filling in `YOUR_STUDY_OR_PROGRAM_LABEL` with your own label):**
+
+`emission://join_study?label=YOUR_STUDY_OR_PROGRAM_LABEL&source=github`
+
```

Comment:
you need to expand this further. Where is the screenshot? Where should they specify this config login link?